grammars/perl.cson

```diff
@@ -721,10 +721,17 @@
     'name': 'string.regexp.replace.perl'
   }
   {
-    'match': '\\b(__FILE__|__LINE__|__PACKAGE__|__SUB__|__DATA__|__END__)\\b'
+    'match': '\\b(__FILE__|__LINE__|__PACKAGE__|__SUB__|__END__)\\b'
     'name': 'constant.language.perl'
   }
   {
+    'begin': '(__DATA__)\\n?'
```

Comment:
Should this start with a `\\b` so that something like `FOO__DATA__` isn't mis-interpreted?